apps/voting/app/src/screens/VoteDetail.js

```diff
@@ -151,139 +179,36 @@ function VoteDetail({ vote, onBack, onVote, onExecute }) {
                       margin-bottom: ${2 * GU}px;
                     `}
                   >
-                    Created By
+                    {open ? 'Current votes' : 'Votes'}
                   </h2>
-                  <div
+                  <SummaryBar
+                    positiveSize={votesYeaVotersSize}
+                    negativeSize={votesNayVotersSize}
+                    requiredSize={supportRequired}
                     css={`
-                      display: flex;
-                      align-items: flex-start;
+                      margin-bottom: ${2 * GU}px;
                     `}
-                  >
-                    <LocalIdentityBadge
-                      connectedAccount={addressesEqual(
-                        creator,
-                        connectedAccount
-                      )}
-                      entity={creator}
-                    />
-                  </div>
+                  />
+                  <SummaryRows
+                    yea={{ pct: yeaPct, amount: yea }}
+                    nay={{ pct: nayPct, amount: nay }}
+                    symbol={tokenSymbol}
+                    connectedAccountVote={connectedAccountVote}
+                  />
+                  {youVoted && <VoteCast vote={vote} />}
                 </div>
-              </div>
-              <div>
-                <h2
-                  css={`
-                    ${textStyle('label2')};
-                    color: ${theme.surfaceContentSecondary};
-                    margin-bottom: ${2 * GU}px;
-                  `}
-                >
-                  {open ? 'Current votes' : 'Votes'}
-                </h2>
-                <SummaryBar
-                  positiveSize={votesYeaVotersSize}
-                  negativeSize={votesNayVotersSize}
-                  requiredSize={supportRequired}
-                  css={`
-                    margin-bottom: ${2 * GU}px;
-                  `}
-                />
-                <SummaryRows
-                  yea={{ pct: yeaPct, amount: yea }}
-                  nay={{ pct: nayPct, amount: nay }}
-                  symbol={tokenSymbol}
-                  connectedAccountVote={connectedAccountVote}
+                <VoteActions
+                  onExecute={handleExecute}
+                  onVoteNo={handleVoteNo}
+                  onVoteYes={handleVoteYes}
+                  vote={vote}
                 />
-                {youVoted && <VoteCast vote={vote} />}
-              </div>
-              <VoteActions
-                onExecute={handleExecute}
-                onVoteNo={handleVoteNo}
-                onVoteYes={handleVoteYes}
-                vote={vote}
-              />
-            </section>
-          </Box>
-        }
-        secondary={
-          <React.Fragment>
-            <Box heading="Status">
-              <Status vote={vote} />
```

Comment:
Now that `VoteInfoBoxes` has replaced this, perhaps we could remove the `<Status>` component code from this view?

Reply:
Of course! I deleted it  later but it slipped in this PR😅. Thanks! :)